Week-3/day6/animating/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE HTML>
+<html>
+   <head>
+      <title>Question and Answer</title>
+      <style>
+   
+         body {
+            font-family: Calibri, sans-serif;
+            font-size: 3em;
+         }
+   
+         #answer {
+            color: black;
+            width: 100px;
+            /* Part of your solution goes here */
+            animation: moveFraction 2s 1s forwards;
+         }
+   
+         /* moveFraction goes here */
```

Comment:
The comment /* moveFraction goes here */ can be removed since the keyframes moveFraction is already defined below it. Keep comments more descriptive than just names of func or any